src/scenes/AcceleratorRouter/Scoring/PhaseScores/ScoreEntry/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
+
+import { Box, Typography, useTheme } from '@mui/material';
+
+import TextArea from 'src/components/common/TextArea';
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { Phase, Score, ScoreValue, getPhaseTruncated, getScoreCategory } from 'src/types/Score';
+
+import ValueControl from './ValueControl';
+
+type ScoringEntryProps = {
+  disabled?: boolean;
+  onChangeValue?: (score: Score, value: ScoreValue) => void;
```

Comment:
Can use just category instead of score in `onChangeValue` and `onChangeQualitative`